src/ipc/dbus/ipc-dbus.cpp

```diff
@@ -136,6 +136,13 @@ bool DBusIPCServiceAdapterBase::handleMessage(const QDBusMessage &dbusMsg, const
     return false;
 }
 
+DBusIPCServiceAdapterBase::~DBusIPCServiceAdapterBase()
+{
+    destroyed(this);
```

Comment:
we should start adding "emit", even if it is void, it makes it easier to read.

Reply:
done